packages/nodes-base/nodes/Orbit/MemberDescription.ts

```diff
@@ -0,0 +1,380 @@
+import {
+	INodeProperties,
+} from 'n8n-workflow';
+
+export const memberOperations = [
+    {
+        displayName: 'Operation',
+        name: 'operation',
+        type: 'options',
+        displayOptions: {
+            show: {
+                resource: [
+                    'member',
+                ],
+            },
+        },
+        options: [
+            {
+                name: 'Delete',
+                value: 'delete',
+                description: 'Delete a member',
+            },
+            {
+                name: 'Get',
+                value: 'get',
+                description: 'Get a member',
+            },
+            {
+                name: 'Get All',
+                value: 'getAll',
+                description: 'Get all members in a workspace',
+            },
+            {
+                name: 'Lookup',
+                value: 'lookup',
+                description: 'Lookup a member or identity by GitHub username',
+            },
+            {
+                name: 'Update',
+                value: 'update',
+                description: 'Update a member',
+            },
```

Comment:
There is a `find` method that we should include, it lets us find a member by one of their github/twitter/email/etc identities. It is now in the swagger docs at https://app.orbit.love/api-docs/index.html.